crates/turborepo-repository/src/package_graph/mod.rs

```diff
@@ -88,8 +88,8 @@ impl PackageInfo {
     }
 }
 
-type PackageKey = String;
-type PackageVersion = String;
+pub type PackageKey = String;
+pub type PackageVersion = String;
```

Comment:
I don't like these type aliases, can we keep them contained to this crate?

Reply:
I didn't want to have the type be `BTreeMap<String, String>`, since that's sort of opaque. Should I just copy them over?